eph-es/install.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+wget -q https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.4.2-linux-x86_64.tar.gz
+wget -q https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.4.2-linux-x86_64.tar.gz.sha512
+shasum -a 512 -c elasticsearch-oss-7.4.2-linux-x86_64.tar.gz.sha512 
+tar -xzf elasticsearch-oss-7.4.2-linux-x86_64.tar.gz --skip-old-files
+
+elasticsearch-7.4.2/bin/elasticsearch &
+
+#wait for elasticserach to start before trying to load templates
+while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8080/)" != "200" ]]; do sleep 1; done
```

Comment:
Since elasticsearch has a healthcheck API, we should probably prefer that when determining when we can start sending traffic https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html#cluster-health-api-request
```suggestion
while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8080/_cluster/health/)" != "200" ]]; do sleep 1; done
```

Reply:
I am not sure that really makes sense here. I am only looking a the return code to see when the app is up. The return code on the health endpoint does not signal anything different than on the base endpoint.